src/loaders.py

```diff
@@ -285,7 +314,11 @@ def get_datagen(self, images_data, annotations, annotations_human_labels, train_
                                    fixed_h=self.dataset_params.fixed_h,
                                    fixed_w=self.dataset_params.fixed_w,
                                    sampler_name=self.dataset_params.sampler_name,
-                                   image_transform=self.image_transform)
+                                   image_transform=self.image_transform,
+                                   # For some reasons AttrDict magic makes something weird
```

Comment:
@i008 I ran into that issue and it turns out that calling it via ['key'] works just fine.

#### FYI:
Quite general version (I think) is now implemented here https://github.com/neptune-ml/open-solution-salt-detection that handles all of different stuff like TTA and stuff.